rct229/rules/section22/section22rule20.py

```diff
@@ -0,0 +1,92 @@
+from rct229.data_fns.table_3_1_3_11_fns import table_3_1_3_11_lookup
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.schema.config import ureg
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import CalcQ
+from rct229.utils.std_comparisons import std_equal
+
+APPLICABLE_SYS_TYPES = [
+    "SYS-7",
+    "SYS-8",
+    "SYS-11.1",
+    "SYS-11.2",
+    "SYS-12",
+    "SYS-13",
+    "SYS-7B",
+    "SYS-8B",
+    "SYS-11B",
+    "SYS-12B",
+    "SYS-13B",
+]
+
+
+class Section22Rule20(RuleDefinitionListIndexedBase):
+    """Rule 20 of ASHRAE 90.1-2019 Appendix G Section 22 (Chilled water loop)"""
+
+    def __init__(self):
+        super(Section22Rule20, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            required_fields={
+                "$": ["weather"],
+                "weather": ["climate_zone"],
+            },
+            each_rule=Section22Rule20.HeatRejectionRule(),
+            index_rmr="baseline",
+            id="22-20",
+            description="The baseline minimum condenser water reset temperature is per Table G3.1.3.11.",
+            list_path="ruleset_model_instances[0].heat_rejections[*]",
+            data_items={"climate_zone": ("baseline", "weather/climate_zone")},
+        )
+
+    def is_applicable(self, context, data=None):
+        rmi_b = context.baseline
```

Comment:
Since you, correctly, didn't set the `rmr_context` above, the context here is at the RMD level. So
```
rmd_b = context.baseline
rmi_b = rmd_b["ruleset_model_instances"][0]
```
You should probably also include
```
required_fields={
    "$": ["ruleset_model_instances"]
}
```
in the initializer above.  Unfortunately, we can't use a required field to ensure the `[0]` ruleset_model_instance exists. You will have to use `assert_` between the rmd_b and rmi_b assignments above:
```
assert_(
    # Should be a non-empty list
    ruleset_model_instance,
    "ruleset_model_instance list is empty"
)
```

Reply:
Thank you so much for catching this up, @charliepnnl.
One remaining question is for the ```required_fields``` in the comment, the problem is ```required_fields``` is also used in the ```HeatRejectionRule``` level,. In other words, we use ```required_fields``` two times, which causes an error. In this case, how we could solve this?

Reply:
I added a couple of new comments. Do those answer your questions here?
BTW: It is perfectly fine for each level to include its own `required_fields`. Generally, the required_fields should go in the lowest level possible.

Reply:
Yes! Fixed.